scripts/dts/extract_dts_includes.py

```diff
@@ -508,16 +551,15 @@ def extract_node_include_info(reduced, root_node_address, sub_node_address,
         y_node = y_sub
 
     if yaml[node_compat].get('use-property-label', False):
-        for yp in y_node['properties']:
-            if yp.get('label') is not None:
-                if node['props'].get('label') is not None:
-                    label_override = convert_string_to_label(
-                        node['props']['label']).upper()
-                    break
+        try:
+            label = y_node['properties']['label']
+            label_override = convert_string_to_label(
+                                    node['props']['label']).upper()
+        except:
```

Comment:
except should always be filtered with an exception type so you don't get an unexpected exception (e.g. OOM, syntax error) and have a hard time debugging.

Reply:
Actually, I as trying to avoid 2 steps check to see if y_node has ['properties']['label'].
If it hasn't, it's not actually an error, we just need to go on with last part of the function.
Would there be another way to do that or am I stuck with 2 steps check?

Reply:
Then I would do "except KeyError" at least.

https://stackoverflow.com/a/15036813/1134134

Reply:
ok, Thanks!
Then maybe the whole script would need a clean up then.
I'll do it in a separate commit